checkstyle/suppressions.xml

```diff
@@ -41,6 +41,6 @@
     />
     <suppress
             checks="(ParameterNumber)"
-            files="(ClusterData).java"
+            files="(ClusterData|ConsumerGroupLagData|ConsumerLagData).java"
```

Comment:
You can disable this check in-place in the code, using comments, e.g.:

```
// CHECKSTYLE:OFF:ParameterNumber
void myMethodWithALotOfParameters(int a, int b, int c, int d, int e, int f, int g, int h, int i, int j) {
}
// CHECKSTYLE:ON:ParameterNumber
```

It's easier to maintain that than these suppressions in this xml file.